extmod/modonewire.c

```diff
@@ -144,7 +144,7 @@ STATIC mp_obj_t onewire_crc8(mp_obj_t data) {
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(onewire_crc8_obj, onewire_crc8);
 
 STATIC const mp_rom_map_elem_t onewire_module_globals_table[] = {
-    { MP_ROM_QSTR(MP_QSTR___name__), MP_ROM_QSTR(MP_QSTR_onewire) },
```

Comment:
Uh, should this be removed? I'm actually not sure how `onewire.py` works if it does `import _onewire as _ow`.

Reply:
removed

Reply:
taking my words back, this should be retained: `_onewire` was registered with `MP_QSTR__onewire` but `_onewire.__name__` was `onewire`...

I'll change it back. It will even save on space because now we only need the `_onewire` string statically, not the `onewire` :)